aiutils/tftools/plholder_management.py

```diff
@@ -0,0 +1,154 @@
+"""This module defines a helper class for managing placeholders in Tensorflow.
+
+The PlholderManager class allows easy management of placeholders including 
+adding placeholders to a Tensorflow graph, producing easy access to the added
+placeholders using a dictionary with placeholder names as keys, create feed 
+dictionary from a given input dictionary, and print placeholders and feed dict 
+to standard output. 
+
+More importantly, the class allows sparse scipy matrices to 
+be passed into graphs (Tensorflow currently allows only dense matrices to be fed
+into placeholders).
+"""
+import tensorflow as tf
+import numpy as np
+import scipy.sparse as sps
+import pdb
+
+
+class PlholderManager():
```

Comment:
Why not PlaceholderManager (it's only 3 more characters).

(same with plholder everywhere)